### PR TITLE
Improved support for libxsmm

### PR DESCRIPTION
**Pull request by benoitsteiner**

### CODE REVIEW

File: tensorflow/workspace.bzl [hunk 1]
```diff
@@ -76,7 +76,6 @@ def tf_workspace(path_prefix = "", tf_repo_name = ""):
   native.new_http_archive(
       name = "libxsmm_archive",
       urls = [
-          # "http://bazel-mirror.storage.googleapis.com/github.com/hfp/libxsmm/archive/1.6.1.tar.gz",
```
Comment (gunan) [inline tensorflow/workspace.bzl]: Let's also add the bazel-mirror one. The link should look like:
http://bazel-mirror.storage.googleapis.com/github.com/hfp/libxsmm/archive/1.6.5.tar.gz

@jart once we make this change, is there a mechanism to automatically mirror the file under bazel-mirror?

Comment (jart) [inline tensorflow/workspace.bzl]: There's no consequences to having this here and the URL not being active, aside from 500ms extra latency, which is amortized out by the eight downloader threads.

My process right now is I run that script you reviewed every few days or so, and whatever GCS URLs exist in the file that aren't currently mirrored, get mirrored.

Comment (jart) [inline tensorflow/workspace.bzl]: Now mirrored: http://bazel-mirror.storage.googleapis.com/github.com/hfp/libxsmm/archive/1.6.5.tar.gz

File: tensorflow/core/kernels/BUILD [hunk 2]
```diff
@@ -2128,7 +2128,10 @@ tf_kernel_library(
 tf_kernel_library(
     name = "matmul_op",
     defines = select({
```
Comment (jart) [inline tensorflow/core/kernels/BUILD]: Is there any reason why we wouldn't want to always enable libxsmm for CPUs that support it?

File: tensorflow/workspace.bzl (hunk 1 above)
Comment (benoitsteiner) [inline tensorflow/workspace.bzl]: Thanks for updating the mirror. It active again.

File: tensorflow/core/kernels/BUILD (hunk 2 above)
Comment (benoitsteiner) [inline tensorflow/core/kernels/BUILD]: We want to do a little more testing before turning it on by default. I'm hoping we'll be able to have it enabled in time for the 1.1 release.

**Review by gunan — APPROVED** [review]

**Review by jart — APPROVED** [review]

**Pull request merged**